docs/guides/flux-v1-automation-migration.md

```diff
@@ -0,0 +1,507 @@
+# Migrating image update automation to Flux v2
+
+"Image Update Automation" is a process in which Flux makes commits to
+your Git repository when it detects that there is a new image to be
+used in a workload (e.g., a Deployment). In Flux v2 this works quite
+differently to how it worked in Flux v1. This guide explains the
+differences and how to port your configuration from v1 to v2.
+
+## Overview of changes between v1 and v2
+
+In Flux v1, image update automation (from here, just "automation") was
+built into the Flux daemon which scanned everything it found in the
+cluster, and updated the Git repository it was syncing (applying to
+the cluster).
+
+In Flux v2,
+
+ - automation is controlled with custom resources, not annotations
+ - ordering images by build time is not supported (see
+   [below](#how-to-use-timestamp-in-image-tags) for what to do
+   instead)
+ - the fields to update are marked explicitly, rather than inferred
+   from annotations.
+
+**Automation is now controlled by custom resources**
```

Comment:
If you make this proper `###`, they show up in the right sidebar on the documentation website under "table of contents".